python/cross_service/textract_comprehend_notebook/TextractAndComprehendNotebook.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f4b62051",
+   "metadata": {},
+   "source": [
+    "This cross-service notebook will walk you through the process of using Textract's DetectDocumenText API to extract text from a JPG/JPEG/PNG file containing text, and then using Comprehend's DetectEntities API to find entities in the extracted text."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93c62ecb",
+   "metadata": {},
+   "source": [
+    "You can run this notebook using either AWS Sagemaker or Jupyter's NBViewer tool."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8496bef4",
+   "metadata": {},
+   "source": [
+    "### To run this notebook using Sagemaker\n",
+    "\n",
+    " 1. Navigate to Amazon Sagemaker from the AWS Console. From the Sagemaker dashboard, select Notebook and then choose \"Notebook instances\".\n",
+    " 2. Select the optional \"Git repositories\" menu, give the notebook instance a name, and choose the option to \"Clone a public Git repository to this notebook instance only\". \n",
+    " 3. In the \"Git repository URL\" box, paste in the URL for this repository (https://github.com/awsdocs/aws-doc-sdk-examples).\n",
+    " 4. Select \"Create notebook instance\".\n",
+    " 5. You will need to add some policies to your Sagemaker role so it can access S3, Textract, and Comprehend. You can add the following policies to your role: `AmazonTextractFullAccess`, `ComprehendFullAccess`, `AmazonS3ReadOnlyAccess`. \n",
+    " 6. After the notebook instance has been created, select it from the list of notebooks. \n",
+    " 7. Choose \"Open Jupyter\", and after the Jupyter notebook has started navigate to the directory containing this notebook and select this notebook to run it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "737428bd",
+   "metadata": {},
+   "source": [
+    "### To run this notebook using NBViewer\n",
+    "\n",
+    "1. Navigate to the NBViewer tool on Jupyter's website (https://nbviewer.jupyter.org/). Paste the URL of this notebook into the text box on the page and select \"Go\".\n",
+    "2. On the navigation bar select the Binder symbol to \"Execute on Binder\". After the notebook build you will be able to run the cells."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96c58983",
+   "metadata": {},
+   "source": [
+    "In order to make use of the Boto3 Python SDK through NBViewer, you may need to configure your AWS credentials. In the code cell below, replace \"KeyID\" with the value of your AWS Key ID and replace \"AccessKey\" with the value of your AWS Secret Access Key. Do not run the following cell if running this notebook in Sagemaker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "faf8e08d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!aws configure set aws_access_key_id \"KeyID\"\n",
+    "!aws configure set aws_secret_access_key \"AccessKey\""
```

Comment:
I'm pretty uneasy about pasting AWS credentials into a website like this, to the point where I decided not to test the NBViewer version. Can you verify whether this is acceptable practice? Alternately, we could forego the NBViewer instructions and present only the SageMaker ones.